src/components/TopNav/TopNav.scss

```diff
@@ -43,6 +43,16 @@ $top-nav-height: 60px;
     }
   }
 
+  &__left {
+    align-items: center;
+    background-color: var(--color-bg);
+    display: flex;
+    flex: 1;
+    height: 60px;
```

Comment:
can utilise `$top-nav-height` here instead of a magic number

Reply:
Good call! I'll make the change.